releases/release-1.29/README.md

```diff
@@ -0,0 +1,118 @@
+| Deprecations and Removals blog published                      | Comms | Thursday 16th November 2023                                                                                                       | week 11  | |
+| Burndown Meetings daily (Tuesday & Thursday over Slack)       | Lead           | Monday 20th November 2023                                                                                                | Week 12  | |
+| **[Test Freeze]**                                             | Branch Manager | [01:00 UTC Wednesday 22nd November 2023 / 18:00 PDT Tuesday 21st November 2023](https://everytimezone.com/s/69f88ada)    | week 12  | |
+| Docs deadline — PRs ready for review                          | Docs Lead          | Tuesday 21st November 2023                                                                                           | Week 12  | |
+| 1.29.0-rc.0 released                                          | Branch Manager     | Tuesday 21st November 2023                                                                                           | week 12  | |
+| release-1.29 branch created                                   | Branch Manager     | Tuesday 21st November 2023                                                                                           | week 12  | |
+| release-1.29 jobs created                                     | Branch Manager     | Tuesday 21st November 2023                                                                                           | week 12  | |
+| Major Themes deadline                                         | Comms              | Tuesday 21st November2023                                                                                            | week 12  | |
+| Start final draft of Release Notes                            | Release Notes Lead | Tuesday 21st November 2023                                                                                           | week 12  | |
+| Release blog ready to review                                  | Comms / Docs |  [01:00 UTC Wednesday 22nd November 2023 / 18:00 PDT Tuesday 21st November 2023](https://everytimezone.com/s/69f88ada)     | week 12  | |
+| Major Themes complete                                         | Release Notes Lead | Tuesday 28th November 2023                                                                                           | week 13  | |
+| Docs complete — All PRs reviewed and ready to merge           | Docs Lead          | Tuesday 28th November 2023                                                                                           | week 13  | |
+| Feature blogs ready to review                                 | Enhancement Owner / SIG Leads | Tuesday 28th November 2023                                                                                | week 13  | |
```

Comment:
Consider moving this earlier, eg to the 17th of November.

Reply:
@sftim -- moved `Feature blogs ready to review` to Week 11 (Nov 17).